### ISSUE TITLE
Uncaught Error: Element ref was specified as a string (editor) but no owner was set.

### ISSUE THREAD

**Issue by MehrnooshHajkhalil** [jrm2k6/react-markdown-editor#55]
I am using this package and before I didn't have any problem. but after updating react with `npm install`, this issue happen. When The page is rendering this component produce three errors and all the page goes blank. I do research but I found nothing useful. I don't know how to fix this. Here is my errors in chrome's console : 

`invariant.js:42 Uncaught Error: Element ref was specified as a string (editor) but no owner was set. You may have multiple copies of React loaded. (details: https://fb.me/react-refs-must-have-owner).
    at invariant (invariant.js:42)
    at coerceRef (react-dom.development.js:8579)
    at reconcileSingleElement (react-dom.development.js:9379)
    at reconcileChildFibers (react-dom.development.js:9477)
    at reconcileChildrenAtPriority (react-dom.development.js:10127)
    at reconcileChildren (react-dom.development.js:10118)
    at finishClassComponent (react-dom.development.js:10254)
    at updateClassComponent (react-dom.development.js:10226)
    at beginWork (react-dom.development.js:10605)
    at performUnitOfWork (react-dom.development.js:12573)`

`react-dom.development.js:8305 The above error occurred in the <MarkdownEditorContent> component:
    in MarkdownEditorContent
    in div
    in MarkdownEditor (created by TopicEditComponent)
    in div (created by TopicEditComponent)
    in div (created by TopicEditComponent)
    in div (created by TopicEditComponent)
    in div (created by TopicEditComponent)
    in form (created by TopicEditComponent)
    in div (created by TopicEditComponent)
    in TopicEditComponent (created by TopicComponent)
    in div (created by TopicComponent)
    in TopicComponent (created by CountryContainer)
    in div (created by CountryContainer)
    in CountryContainer (created by Connect(CountryContainer))
    in Connect(CountryContainer) (created by RouterContext)
    in div (created by CoreLayout)
    in div (created by CoreLayout)
    in CoreLayout (created by RouterContext)
    in RouterContext (created by Router)
    in Router (created by Routes)
    in Routes
    in Provider
Consider adding an error boundary to your tree to customize error handling behavior.
You can learn more about error boundaries at https://fb.me/react-error-boundaries.`

`react-dom.development.js:1345 Uncaught Error: A cross-origin error was thrown. React doesn't have access to the actual error object in development. See https://fb.me/react-crossorigin-error for more information.
    at Object.invokeGuardedCallbackDev (react-dom.development.js:1345)
    at invokeGuardedCallback (react-dom.development.js:1195)
    at performWork (react-dom.development.js:12800)
    at batchedUpdates (react-dom.development.js:13244)
    at performFiberBatchedUpdates (react-dom.development.js:1646)
    at stackBatchedUpdates (react-dom.development.js:1637)
    at batchedUpdates (react-dom.development.js:1651)
    at Object.batchedUpdatesWithControlledComponents [as batchedUpdates] (react-dom.development.js:1664)
    at dispatchEvent (react-dom.development.js:1874)`

**Comment by jaywcjlove** [comment]
The error is caused by this： https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs

**Comment by gerryl1991** [comment]
have you fixed this? I got the same error, while working on a modal using antd. Don't know how to fixed it.

**Comment by jrm2k6** [comment]
Hi,
Sorry for not answering before. I didn't receive any notifications in my emails.
I apologize.

@MehrnooshHajkhalil @gerryl1991 What version of react are you using?

**Comment by jrm2k6** [comment]
Thanks. Will check.

**Comment by dsacramone** [comment]
I'm getting this too, any resolution? I am using:
react: 16.2.0
react-dom: 16.2.0

I was seeing it on 16, then upgraded, still seeing it.

**Comment by jrm2k6** [comment]
Yes I didn't get to it yet. Let me see if I can fix the issue this weekend.

**Comment by jrm2k6** [comment]
The issue right now is that Reflux is not compatible with React 16 as far as I can tell https://github.com/reflux/refluxjs/issues/537

**Comment by jrm2k6** [comment]
@dsacramone @gerryl1991 @jaywcjlove @MehrnooshHajkhalil Can you try to install the package using: `npm install --save react-markdown-editor@beta`. That one should work with React 16.
